pkg/skaffold/runner/runner.go

```diff
@@ -60,8 +60,8 @@ type SkaffoldRunner struct {
 	builds       []build.Artifact
 }
 
-// NewForConfig returns a new SkaffoldRunner for a SkaffoldConfig
-func NewForConfig(opts *config.SkaffoldOptions, cfg *latest.SkaffoldConfig) (*SkaffoldRunner, error) {
+// NewForConfig returns a new SkaffoldRunner for a SkaffoldPipeline
+func NewForConfig(opts *config.SkaffoldOptions, cfg *latest.SkaffoldPipeline) (*SkaffoldRunner, error) {
```

Comment:
I think its going to be a lot more confusing if functions like `NewForConfig` takes in a `latest.SkaffoldPipeline`

Reply:
We can always rename `NewForConfig` to `NewForPipeline`